compiler/ast.nim

```diff
@@ -196,6 +196,7 @@ type
     nkTypeOfExpr,         # type(1+2)
     nkObjectTy,           # object body
     nkTupleTy,            # tuple body
+    nkTupleClassTy,       # tuple type class
```

Comment:
I think I did that—though I updated it in the  form of a squash rebase, so it might've gone unnoticed.